update description (#1)
DevExpress-Examples/asp-net-web-forms-grid-custom-function-filter-criteria-operators: Readme.md

# Grid View for ASP.NET Web Forms - How to apply custom function filter criteria operators
<!-- run online -->
**[[Run Online]](https://codecentral.devexpress.com/t546944/)**
<!-- run online end -->

This example demonstrates how to use a [CriteriaPatchBase](https://supportcenter.devexpress.com/ticket/details/t320172/how-to-traverse-through-and-modify-the-criteriaoperator-instances) class descendant to apply a [custom function criteria operator](https://docs.devexpress.com/eXpressAppFramework/113480/filtering/in-list-view/custom-function-criteria-operators).

![Custom function filter criteria operator](customFunctionCriteriaOperator.png)

## Overview

> Use the grid's [FilterExpression](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridBase.FilterExpression) property to replace a custom function operator with an actual one. For auto filter row, handle the grid's server-side [ProcessColumnAutoFilter](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView.ProcessColumnAutoFilter) event as follows:

Reviewer: What do you mean by `with an actual one`? A custom function operator is still an operator, so what is this 'actual one' you mention?

`For auto filter row` Do you mean "For the auto filter row"?

Author: ```suggestion
Assign a custom function operator to the grid's [FilterExpression](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridBase.FilterExpression) property. For the auto filter row, handle the grid's server-side [ProcessColumnAutoFilter](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView.ProcessColumnAutoFilter) event as follows:
```
